libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -792,7 +792,7 @@ AP_AHRS_DCM::drift_correction(float deltat)
     // flat, but still allow for yaw correction using the
     // accelerometers at high roll angles as long as we have a GPS
     if (AP_AHRS_DCM::use_compass()) {
-        if (have_gps() && is_equal(gps_gain,1.0f)) {
+        if (have_gps() && is_equal<float>(gps_gain,1.0f)) {
```

Comment:
_gps_gain_ (and _ _L1_xtrack_i_gain_) is an AP_Float and _1.0f_ (and _ _L1_xtrack_i_gain_prev_) is a float, there's no int in these cases.